.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,7 +2,7 @@
 
 Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
```

Comment:
This sounds like PRs have to have an associated Issue. Also, just for type defect. Both intentional?

Reply:
Wording changed. This, among others, is only applicable when a repo does not set up its own templates so a more generic language suits better.